packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -124,48 +167,22 @@ export class InsuredBridgeL2Client {
     });
   }
 
-  async getFundsDepositedEvents(eventSearchOptions: EventSearchOptions): Promise<EventData[]> {
+  async getBridgeDepositBoxEvents(eventSearchOptions: EventSearchOptions, eventName: string): Promise<EventData[]> {
     const eventsData = await getEventsForMultipleProviders(
       [this.l2Web3].concat(this.fallbackL2Web3s),
       getAbi("BridgeDepositBox"),
       this.bridgeDepositAddress,
-      "FundsDeposited",
+      eventName,
       eventSearchOptions
     );
     if (eventsData.missingEvents.length > 0) {
-      const errorMessage = `L2 RPC endpoints disagree about L2 contract events, please manually investigate. L2 rpcs are described in NODE_URL and RETRY_CONFIG environment variables.`;
```

Comment:
I also refactored how we make L2 calls to have a general `getBridgeDepositBoxEvents` that uses @nicholaspai 's `getEventsForMultipleProviders`. previously this was repeated for each internal call. now that there are 3 of them we should have a common function for doing this where you pass in the `eventName`